Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -121,10 +130,13 @@ class SettingsContentViewController: UIViewController, WKNavigationDelegate, The
         let config = LegacyTabManager.makeWebViewConfig(isPrivate: true, prefs: nil)
         config.preferences.javaScriptCanOpenWindowsAutomatically = false
 
-        let webView = WKWebView(
-            frame: CGRect(width: 1, height: 1),
-            configuration: config
-        )
+        let screenWidth = UIScreen.main.bounds.width
+        let screenHeight = UIScreen.main.bounds.height
+
+        let webViewFrame = CGRect(x: 0, y: 0, width: screenWidth, height: screenHeight)
+        let webView = WKWebView(frame: webViewFrame, configuration: config)
```

Comment:
All this code can be removed since we are using auto layout and we could set the frame of the webView like this: 
       let webView = WKWebView(
            frame: .zero,
            configuration: config
        )